widgets/src/c3/Gauge.js

```diff
@@ -0,0 +1,49 @@
+(function (root, factory) {
+    if (typeof define === "function" && define.amd) {
+        define(["./Common"], factory);
+    } else {
+        root.Gauge = factory(root.Common);
+    }
+}(this, function (Common) {
+    function Gauge(target) {
+        Common.call(this);
+
+        this._class = "c3_gauge";
```

Comment:
New convention - should be "c3_Gauge" - IOW the class name should match the path c3/Gauge with an underbar instead of the "/".  I will be using this to "locate" a persisted widgets JS file so needs to match exactly...